velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -1500,5 +1510,22 @@ TEST_F(Re2FunctionsTest, split) {
   assertEqualVectors(expected, result);
 }
 
+TEST_F(Re2FunctionsTest, testParseSubstrings) {
```

Comment:
drop 'test'; please, review coding style guidelines and make sure the code is compliant